lib/fog/azurerm/docs/storage.md

```diff
@@ -176,42 +166,119 @@ result = container.destroy
 puts "#{result}"
 ```
 
-### Metadata
+## Upload a local file as a blob
+```ruby
+new_blob = azure_storage_service.blobs.get('<Container name>', '<Blob name>').create(file_path: '<file path>')
```

Comment:
Why are you getting blob object and then creating it again ?
Can't we do this as follows?

```
new_blob = azure_storage_service.blobs..create(
  name: 'myblob', 
  container_name: 'mycontainer', 
  file_path: '<file path>')
```